src/patternfly/components/Table/table.scss

```diff
@@ -750,33 +757,9 @@
 
 // - Table check
 .#{$table}__check {
-  --#{$table}--cell--PaddingInlineStart: var(--#{$table}__check--PaddingInlineStart);
-  --#{$table}--cell--PaddingInlineEnd: var(--#{$table}__check--PaddingInlineEnd);
-
-  vertical-align: top;
-
-  .#{$check} {
-    --#{$check}__label--FontSize: var(--#{$table}--cell--FontSize);
-    --#{$check}__label--LineHeight: var(--#{$table}--cell--LineHeight);
-  }
-
+  .#{$check},
   .#{$radio} {
-    --#{$radio}__label--FontSize: var(--#{$table}--cell--FontSize);
-    --#{$radio}__label--LineHeight: var(--#{$table}--cell--LineHeight);
-  }
-
-  thead & {
-    vertical-align: bottom;
-  }
-
-  .#{$check}.pf-m-standalone,
-  .#{$radio}.pf-m-standalone {
-    thead & {
-      display: table-cell;
-      height: auto;
-      line-height: 1;
-      vertical-align: baseline;
-    }
```

Comment:
Special styling for `__check` is no longer necessary as its height has been standardized to `table-_FontSize * table__LineHeight` or `21px` resulting in vertical alignment with all other elements.